python/LICENSE.txt

```diff
@@ -0,0 +1,1085 @@
+
```

Comment:
We could make this part of the release build so that we only push this to the release and not drift on changes of the Cargo.toml (or need to remember to update it)

Reply:
That's a good idea, I will remove it from the PR.